CHANGELOG.md

```diff
@@ -20,6 +20,9 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ### Changed
 
+- Upgrade all use of `go.opentelemetry.io/otel/semconv` to use `v1.23.1`. (#4754)
+- Update transformations in `go.opentelemetry.io/otel/exporters/zipkin` to follow `v1.19.0` version of the OpenTelemetry specification.
+  It handles spans following semantic convention `v1.22.0` version or greater. (#4754)
```

Comment:
**Notice**

These changes follows https://github.com/open-telemetry/opentelemetry-specification/pull/3713 (released in https://github.com/open-telemetry/semantic-conventions/releases/tag/v1.22.0) which are not backwards compatible.

The problem is that updated mappings will work great only for code that is uses semver >= v1.22.0. 

If an instrumentation library is using e.g. v1.20.0 semver schema then it may now miss the mapping. E.g.  `otelhttp`, `otelgrpc` add `net.sock.peer.addr` attribute and these instrumentation libraries will no longer nicely integrate with the exporter.

Brainstorming: We could probably handle some of changes like this in a backwards compatible way if we would do the schema translation for the spans that are coming to the exporter. But this would require https://github.com/orgs/open-telemetry/projects/64.

Reply:
People can always use previous version of the exporter if they do not like the translation changes.

Reply:
SIG meeting: We can have a fallback to handle previous semconv.

Reply:
I found more breaking changes.
I created https://github.com/open-telemetry/opentelemetry-specification/issues/3793 and https://github.com/open-telemetry/opentelemetry-specification/pull/3794.

Marking as blocked by specification.

Reply:
I got 5 approvals in https://github.com/open-telemetry/opentelemetry-specification/pull/3794 so far so I decided to implement as I proposed.

Reply:
The PR is merged. We can wait for a spec release, but I think it is safe to review and merge (if approved).